specification/sql/resource-manager/Microsoft.Sql/preview/2024-05-01-preview/InstancePoolOperations.json

```diff
@@ -0,0 +1,272 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the specified instance pool operation.",
+            "schema": {
+              "$ref": "#/definitions/InstancePoolOperation"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 ManagedInstanceStoppingOrStopped - Conflicting operation submitted while instance is in stopping/stopped state\n\n * 400 ManagedInstanceStarting - Conflicting operation submitted while instance is in starting state\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription.\n\n * 404 ResourceNotFound - The requested resource was not found.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Gets the instance pool management operation": {
+            "$ref": "./examples/GetInstancePoolOperation.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "InstancePoolOperation": {
+      "description": "A instance pool operation.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/InstancePoolOperationProperties",
+          "description": "Resource properties.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "InstancePoolOperationListResult": {
+      "description": "The response to a list instance pool operations request",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/InstancePoolOperation"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "InstancePoolOperationProperties": {
+      "description": "The properties of a instance pool operation.",
+      "type": "object",
+      "properties": {
+        "instancePoolName": {
+          "description": "The name of the instance pool the operation is being performed on.",
+          "type": "string",
+          "readOnly": true
+        },
+        "operation": {
+          "description": "The name of operation.",
+          "type": "string",
+          "readOnly": true
+        },
+        "operationFriendlyName": {
+          "description": "The friendly name of operation.",
+          "type": "string",
+          "readOnly": true
+        },
+        "percentComplete": {
```

Comment:
>percentComplete [](http://example.com/codeflow?start=9&length=15)

operationCompletedPercentage?

Reply:
Same as for Managed instance, keeping request/response objects as similar as possible